src/environments/classic_control/mountain_car.jl

```diff
@@ -70,17 +70,18 @@ function MountainCarEnv(; T = Float64, continuous = false, seed = nothing, kwarg
     else
         params = MountainCarEnvParams(; T = T, kwargs...)
     end
+    action_space = continuous ? ContinuousSpace(-T(1.0), T(1.0)) : DiscreteSpace(3)
     env = MountainCarEnv(
         params,
-        continuous ? ContinuousSpace(-T(1.0), T(1.0)) : DiscreteSpace(3),
+        action_space,
         MultiContinuousSpace(
             [params.min_pos, -params.max_speed],
             [params.max_pos, params.max_speed],
         ),
         zeros(T, 2),
         1,
         false,
-        0,
+        rand(action_space),
```

Comment:
The corresponding field here is `t`, which means `timestamp`. So I think we don't need to change it here. And it will fix the [error](https://travis-ci.com/github/JuliaReinforcementLearning/ReinforcementLearningEnvironments.jl/builds/169337756#L3465)
```suggestion
        0,
```